heron/config/src/yaml/conf/aurora/scheduler.yaml

```diff
@@ -8,4 +8,10 @@ heron.class.launcher:                        com.twitter.heron.scheduler.aurora.
 heron.directory.sandbox.java.home:           /usr/lib/jvm/java-1.8.0-openjdk-amd64/
 
 # Invoke the IScheduler as a library directly
-heron.scheduler.is.service:                  False
\ No newline at end of file
+heron.scheduler.is.service:                  False
+
+# Override the default Heron Aurora template
+#heron.scheduler.job.template:                heron.aurora
+
+# Override the default directory for the Aurora template
+#heron.scheduler.job.template.dir:            /etc/heron
```

Comment:
I wanted to put something in this file to indicate that it could be set to a different, fixed directory. The Aurora Context file already does the fall back if the tag is not defined. I thought that having it here but commented out made sense since it would easy to simply uncomment the line and change the directory to make use of it.